run-checks

```diff
@@ -4,9 +4,9 @@ export LANG=C.UTF-8
 export LANGUAGE=en
 
 if command -v goctest >/dev/null; then
-    goctest="goctest"
+    goctest="goctest ./..."
 else
-    goctest="go test"
+    goctest="go test ./..."
 fi
```

Comment:
not sure I ever used goctest or if anyone is using it, but if they do they can prob. run the test suite themselves